ufo/user_test.py

```diff
@@ -246,40 +240,42 @@ def testAddUsersPostManualHandler(self, mock_add):
     response = self.client.post(flask.url_for('add_user'), data=data,
                                 follow_redirects=False)
 
+    matched_count = models.User.query.filter_by(email=FAKE_EMAILS_AND_NAMES[0]['email']).count()
```

Comment:
It would be nice to assert a bit more on the manual add's execution path, rather just a count of something was added.  For example, the data could have user_name and user_email and then test if those are populated into the database correctly.

 user_name = flask.request.form.get('user_name')
 user_email = flask.request.form.get('user_email')